nms/app/context/FEGGatewayContext.tsx

```diff
@@ -28,7 +28,7 @@ export type FEGGatewayContextType = {
   setState: (
     key: GatewayId,
     val?: MutableFederationGateway,
-    newState?: Record<string, FederationGateway>,
```

Comment:
I would much rather have a separate `updateTier` method. Or maybe even and `updateGateway` method to keep it consistent with LTE.